test/integration/step_defs_test.go

```diff
@@ -103,7 +105,7 @@ func (f *feature) getGoscaleioClient() (client *goscaleio.Client, err error) {
 	if f.arrays == nil {
 		fmt.Printf("Initialize ArrayConfig from %s:\n", configFile)
 		var err error
-		f.arrays, err = f.getArrayConfig()
+		f.arrays, err = f.getArrayConfig(configFile)
```

Comment:
In this file you now mix configFile and configurationFile.

Reply:
Here we are calling the function, hence passing the path of the config file.

Reply:
We are consistent with passing configFile and zoneConfigFile paths as arguments when calling the getArrayConfig or readConfigFile functions.